front/lib/api/oauth.ts

```diff
@@ -244,6 +244,25 @@ const PROVIDER_STRATEGIES: Record<
       return getStringFromQuery(query, "state");
     },
   },
+  zendesk: {
+    setupUri: (connection) => {
+      const scopes = ["tickets:read", "hc:read"];
+      return (
+        `https://d3v-dust.zendesk.com/oauth/authorizations/new?` +
```

Comment:
this one seems like a non production URL? is `d3v-dust` the namespace we want to use to go to production?